mottak/src/main/kotlin/no/nav/melosys/soknadmottak/kvittering/KvitteringService.kt

```diff
@@ -37,6 +39,7 @@ class KvitteringService(private val korrespondanseService: KorrespondanseService
         } catch (t: Throwable) {
             throw IntegrasjonException("Kunne ikke kvittere for arkiv '$arkivRef'", t)
         }
+        Metrics.counter(KVITTERING_SENDT).increment()
```

Comment:
Disse counterene bør initialiseres først - typisk ved oppstart av applikasjonen. Hvis ikke vil telling alltid være en for lav 🙂

Reply:
For å utdype det litt; `Metrics.counter(KVITTERING_SENDT)` vil sette telleren til 0, som prometheus vil plukke opp ved oppstart.
`Metrics.counter(KVITTERING_SENDT).increment()` vil sette telleren til 0 og deretter umiddelbart sette den til 1. Som vil gjøre at prometheus mest sannsynlig kun vil lese telleren som 1, og ikke få med seg at den har vært 0 tidligere.

I Grafana teller vi jo antall ganger en teller har blitt inkrementert, og med siste eksempelet så er første verdien prometheus vil fange opp `1`. Som videre fører til at vi vil mangle én verdi per nye instans som har eksistert av `melosys-soknad-mottak` 🙂